### ISSUE TITLE
Support library for target framework .net 8

### ISSUE THREAD

**Issue by eoehen** [scriban/scriban#546]
.net 7 is Out of support versions.

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

![image](https://github.com/scriban/scriban/assets/49316770/cbb4e824-c46f-4335-84ca-ebac2bcb8c62)

**Comment by thirstyape** [comment]
Yes, this seems like an odd point to leave the additional target at. As .NET Standard is supported wouldn't it be best to have the additional targets on supported versions of .NET?

**Comment by xoofx** [comment]
> Yes, this seems like an odd point to leave the additional target at. As .NET Standard is supported wouldn't it be best to have the additional targets on supported versions of .NET?

When you target .NET 7.0, your library is automatically supported by higher version. There is usually no real point to target higher versions, unless you start to use new API.

Anyway, I have updated Scriban to `net8.0` by commit fc29ec649adbdfe194c9fc3ab8fbd07a4bf2b34e

Should be available in 7.0 version
